packages/core/src/utilities/createSigmoidRGBTransferFunction.ts

```diff
@@ -0,0 +1,48 @@
+import vtkColorTransferFunction from '@kitware/vtk.js/Rendering/Core/ColorTransferFunction';
+import { VOIRange } from '../types';
+
+/**
+ * A utility that can be used to generate an RgbTransferFunction
```

Comment:
I guess the documentation is wrong here, it should say create Sigmoid RGB? also maybe can you add a little bit of information about why one might prefer this sigmoid one compared to linear?